backend/core/helpers.py

```diff
@@ -24,3 +24,15 @@ def get_first_and_last_names_from_business_email(email):
     ).groups()
 
     return first_name.capitalize(), last_name.capitalize()
+
+
+def add_users_to_team(team, email_addresses):
```

Comment:
Also thinking we could probably add this to the form. And use it in the same fashion we call `form.save()` for model forms. I don't see the need for a separate "helper" function for this yet.